adamsjr8576/final-countdown-svelte
# Testing

### What does this PR do?
- This PR adds testing files to each component
- Adds a snapshot for each component
- tests user functionality when applicable and possible
- tests apiCalls
 - uses testing-library/svelte with jest to test components

### How to test?
 - pull down branch
 - run `npm test`
 - see passing tests

### Issues:
 - I was not able to figure out how to mock out and test the writable items coming from the store - this made it pretty difficult to fully test some of the components.

### Notes:
I installed jest, enzyme and testing-library/svelte as well as babel to configure the tests. I ended up not using enzyme so should probably uninstall that.
